### DISCUSSION TITLE
Love the project

### DISCUSSION THREAD

**Discussion by juanviamonte** [Cerlancism/chatgpt-subtitle-translator#62] [General]
Just question around pricing ? have you guys done any research on average translation cost with ChatGPT?  for a video of 5 mins, 10 mins, 20 mins, etc sort of thing? thanks a lot

**Comment by Cerlancism** [comment]
You can get a rough estimate by using a free translation service, like copying and pasting the `srt` text into Google Translate, do note it also has character count limits. 

Then, take the original `srt` text (input) and the Google Translated text (output), and paste them into 
<https://platform.openai.com/tokenizer>
to obtain the respective token counts. 

Use these token counts to calculate costs on 
<https://openai.com/pricing#LandingPricingUnitConverter-0>

Since GPT, being a large language model (LLM), produces non-deterministic outputs, consider including at least an additional 20% buffer for retries, as mentioned in the `--batch-sizes` section of 
<https://github.com/Cerlancism/chatgpt-subtitle-translator?tab=readme-ov-file#cli>
